internal/storage/sqlite/sqlite.go

```diff
@@ -0,0 +1,220 @@
+package sqlite
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/popvaleks/go_todo_list/internal/models"
+	"github.com/popvaleks/go_todo_list/internal/storage"
+	"strconv"
+)
+
+type Storage struct {
+	db *sql.DB
+}
+
+func New(dbPath string) (*Storage, error) {
+	const op = "storage.sqlite.New"
+
+	db, err := sql.Open("sqlite3", dbPath)
+	if err != nil {
+		return nil, fmt.Errorf("%s: %w", op, err)
+	}
+
+	stmt, err := db.Prepare(`
+	CREATE TABLE IF NOT EXISTS scheduler(
+		id INTEGER PRIMARY KEY,
+		date CHAR(8) NOT NULL DEFAULT "",
+		title TEXT NOT NULL DEFAULT "",
+		comment TEXT NOT NULL DEFAULT "",
+		repeat  CHAR(128) NOT NULL DEFAULT "");
+	`)
+	if err != nil {
+		return nil, fmt.Errorf("%s: %w", op, err)
```

Comment:
Лучше писать суть ошибки, чтобы сразу понять, где она возникла.
```suggestion
		return nil, fmt.Errorf("cannot create table scheduler: %w", err)
```